demisto_sdk/commands/test_content/timestamp_replacer.py

```diff
@@ -10,9 +10,10 @@
 from typing import List, Union
 
 from dateparser import parse
-from mitmproxy import ctx, flow
+from mitmproxy import ctx
+from mitmproxy.addonmanager import Loader
 from mitmproxy.addons.serverplayback import ServerPlayback
-from mitmproxy.http import HTTPRequest
+from mitmproxy.http import HTTPFlow, HTTPRequest
```

Comment:
What's the reason for replacing `flow.Flow` with `HTTPFlow`?

Reply:
Because that's the one it actually uses
![image](https://user-images.githubusercontent.com/41257953/104565453-f99d2880-5654-11eb-8523-8293714a6dcf.png)